swift-ticketing/test/swift_ticketing/fixtures.clj

```diff
@@ -1,20 +1,53 @@
 (ns swift-ticketing.fixtures
   (:require
-   [next.jdbc :as jdbc]
-   [honey.sql :as sql]
-   [swift-ticketing.core :refer [create-connection-pool]]
+   [com.stuartsierra.component :as component]
+   [swift-ticketing.core :refer [swift-ticketing-system]]
    [swift-ticketing.factory :as factory]
    [swift-ticketing.config :refer [read-test-config]]
-   [swift-ticketing.migrations :as migrations]))
+   [swift-ticketing.migrations :as migrations]
+   [next.jdbc :as jdbc]
+   [honey.sql :as sql]))
 
 (def test-env
-  (let [db-config (:database (read-test-config))]
-    {:db-spec (create-connection-pool db-config)}))
+  {:test-user-id nil
+   :db-spec nil})
+
+(def swift-ticketing-test-system
+  (swift-ticketing-system (read-test-config)))
+
+(defn start-test-system []
+  (alter-var-root #'swift-ticketing-test-system component/start))
```

Comment:
@neenaoffline is this the right way to start a component process? Seems very unclojury to me 😱